Presentation/Slides-day-1.Rmd

```diff
@@ -0,0 +1,534 @@
+}
+```
+
+## Overview
+
+- At the end of this course, you should be able to 
+  - Calculate point estimates and their standard errors with survey data 
+      - Proportions, totals, and counts
+      - Means, quantiles, and ratios
+  - Perform t-tests and chi-squared tests
+  - Fit regression models
+  - Specify a survey design in R to create a survey object
+  
+- We will not be going over the following but provide some resources at the end
+  - Weighting (calibration, post-stratification, raking, etc.)
+  - Survival analysis
+  - Nonlinear models
+
+
+
+---
+## Overview: Course Roadmap
+
+- Get familiar with RStudio Cloud with a warm-up exercise using the tidyverse (today)
+
+- Introduce the survey data we'll be using in the course (today)
+
+- Analysis of categorical data with time for practice (today)
+
+- Analysis of continuous data with time for practice (day 2)
+
+- Survey design objects, constructing replicate weights, and creating derived variables (day 3)
+
+
+---
+## Logistics
+
+- We will be using RStudio Cloud today to ensure everyone has access
+
+    - Sign-up for a free RStudio Cloud account 
+    - Access the project and files via link in email and Zoom chat
+    - Click "START" to open the project and get started
+    - Rstudio Cloud has the same features and appearance as RStudio for ease of use
+
+- All slides and code are available on GitHub: https://github.com/tidy-survey-r/tidy-survey-short-course
+
+???
+Github repo is for future reference, all material on RStudio cloud
+
+---
+## Intro to RStudio Cloud: Penguins!!
+
+- Using `palmerpenguins` data for warm-up exercises
+
+- Data were collected and made available by Dr. Kristen Gorman and the Palmer Station, Antarctica LTER, a member of the Long Term Ecological Research Network.
+
+- Access data through `palmerpenguins` package https://github.com/allisonhorst/palmerpenguins/
+
+
+####If you are using your own RStudio environment:
+- Make sure you have `tidyverse`, `here`, and `palmerpenguins` installed
+
+```{r inst_packages, error=FALSE, warning=FALSE, eval=FALSE}
+# Run package installation if you don't have these packages already
+# As a reminder, installing takes package from internet to your computer 
+# and only needs to be done once, not each session
+
+install.packages(c("tidyverse", "here", "palmerpenguins"))
+```
+
+---
+## Intro to RStudio Cloud: Penguins!!
+
+- Load `tidyverse`, `here`, and `palmerpenguins`
+
+- Look at the penguins dataset using `glimpse`
+
+```{r load_pack1, error=FALSE, warning=FALSE}
+library(tidyverse) # for tidyverse
+library(here) # for file paths
+library(palmerpenguins) # for warm-up data
+glimpse(penguins)
+```
+
+---
+## Warm-up Exercises: WarmUpExercises.Rmd
+
+- <b>Let's open RStudio cloud and do some warm-up examples</b>
+
+- Explore the penguins data
+  - How many penguins of each species are there? <i>We will do this one together</i>
+  - How many penguins of each species and sex are there? Hint: use `count`
```

Comment:
Should we add another exercise here?  They would only be doing one on their own.  Maybe add in a mean or something?

Reply:
I was trying to contain to categorical analysis but a mean based exercise could still help with some tidyverse syntax on the mind. Thoughts?